src/cluecode/finder.py

```diff
@@ -59,6 +59,23 @@ def logger_debug(*args):
 """
 
 
+def check_if_line_can_contain_urls(line):
+    """
+    Returns true if the line is a likely candidate to include urls.
+    """
+    url_keyword_list = ['://', 'http', 'https', 'www', 'git', 'ftp', 'ftps',
```

Comment:
You could make this a tuple instead and please do not suffix your variable name with `_list`